src/theme/components/heading.ts

```diff
@@ -44,20 +44,79 @@ export const headings = {
     ...margins,
     width: "auto",
   },
+  display1: {
+    fontWeight: "heading.display1",
+    letterSpacing: "1px",
+    lineHeight: "1.05",
+    ...margins,
+    width: "auto",
+  },
+  heading1: {
+    fontWeight: "heading.heading1",
+    letterSpacing: "1px",
+    lineHeight: "1.05",
+    ...margins,
+    width: "auto",
+  },
+  heading2: {
+    fontWeight: "heading.heading2",
+    letterSpacing: "1px",
+    lineHeight: "1.15",
+    ...margins,
+    width: "auto",
+  },
+  heading3: {
+    fontWeight: "heading.heading3",
+    letterSpacing: "1px",
+    lineHeight: "1.15",
+    ...margins,
+    width: "auto",
+  },
+  heading4: {
+    fontWeight: "heading.heading4",
+    lineHeight: "1.2",
+    ...margins,
+    width: "auto",
+  },
+  heading5: {
+    fontWeight: "heading.heading5",
+    lineHeight: "1.2",
+    ...margins,
+    width: "auto",
+  },
+  heading6: {
+    fontWeight: "heading.heading6",
+    lineHeight: "1.2",
+    ...margins,
+    width: "auto",
+  },
 };
 
 // Styles for different visual variants
 const variants = {
-  h1: headings.h1,
-  h2: headings.h2,
-  h3: headings.h3,
-  h4: headings.h4,
-  h5: headings.fallback,
-  h6: headings.fallback,
-  primary: headings.h1,
-  secondary: headings.h2,
-  tertiary: headings.h3,
-  callout: headings.h4,
+  one: headings.one,
+  two: headings.two,
+  three: headings.three,
+  four: headings.four,
+  five: headings.fallback,
+  six: headings.fallback,
+  primary: headings.one,
+  secondary: headings.two,
+  tertiary: headings.three,
+  callout: headings.four,
```

Comment:
Can you group the new headings with the deprecated ones and the new sizes with the deprecated ones and add comments for each group?